src/pages/LoadingPage/LoadingBar/index.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from "@emotion/styled";
+import { palette } from "@leafygreen-ui/palette";
+import { size } from "constants/tokens";
+
+const { gray, green } = palette;
+interface LoadingBarProps {
+  progress: number;
+  indeterminate: boolean;
+}
+const LoadingBar: React.FC<LoadingBarProps> = ({
+  progress,
+  indeterminate = false,
+}) => (
+  <Container>
+    <Bar progress={indeterminate ? 100 : progress} />
+  </Container>
+);
+
+const Container = styled.div`
+  position: relative;
+  height: ${size.s};
```

Comment:
I think this is one of those situations where `size.xs` is too small and `size.s` is too big. How about setting it to `12px`?